app/styles/card.css

```diff
@@ -0,0 +1,53 @@
+/* INFO: not adding width/max-width to let it decicde by layout */
+:root {
+  --border-color: #00000040;
+  /* TODO: get primary color from variables.css, once primary color is decided  */
+  --primary: #1d1283;
+}
+.card {
+  max-width: 41rem;
+  padding: 1.5rem 1rem;
+  border: 1px solid var(--border-color);
+  border-radius: 3rem;
+  box-shadow: 3px 3px 3px var(--border-color);
+  text-align: center;
+}
+
+.card__header {
+  margin-bottom: 0.25rem;
+}
+
+.card__title {
+  color: var(--primary);
+}
+
+.card__description--wrapper {
+  margin-bottom: 0.25rem;
+}
+
+.card__description {
+  display: -webkit-box;
+  -webkit-line-clamp: 4;
+  -webkit-box-orient: vertical;
+  text-overflow: ellipsis;
+  overflow: hidden;
+  font-weight: 400;
+  font-size: 1.25rem;
+}
+
+.card__actions {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.card__button {
+  color: white;
```

Comment:
Can we make use of css variables here instead of hard coding `white` value in color?

Reply:
Possible, but waiting for AO to configure global variables.